portal-web/docroot/html/portal/api/jsonws/action.jsp

```diff
@@ -349,6 +349,17 @@ String signature = ParamUtil.getString(request, "signature");
 						</aui:field-wrapper>
 
 					<%
+					}
+					else if (methodParameterTypeClassName.startsWith("com.")) {
```

Comment:
@SpencerWoo Wouldn't this break if we wanted to add a parameter whose path starts with "org" or something similar? I may be misunderstanding what this is doing though.

Reply:
@jonathanmccann 
Here we're creating different GUI input fields based on the _expected input types_.

For the line here we're looking to trigger when the input type is an object so we're using "com.". 
 The parameter input content can be anything but we're trying to match object parameter types.

However if there are JSONWS methods that takes object parameters but don't start with "com." as a type then it's true that we wouldn't interpret them as an object and the else condition would execute giving the current behavior via GUI.  If this is the case I'd like to talk about it but from looking around at the JSONWS methods I thought "com." was the best to match object parameter types.